src/current/_includes/v23.1/backups/object-dependency.md

```diff
@@ -0,0 +1,7 @@
+When you back up a table, the backup will include all of its dependent objects, but the backup will not include any dependent tables. For example, if you back up [view]({% link {{ page.version.version }}/views.md %}) `v` that depends on table `t`, it will only back up `v`, not `t`. When you try to restore `v`, the restore will fail because the referenced table is not present in the backup. Alternatively, you can pass [`skip_missing_views`]({% link {{ page.version.version }}/restore.md %}#skip-missing-views) with {% if page.name == "restore.md" %} `RESTORE` {% else %} [`RESTORE`]({% link {{ page.version.version }}/restore.md %}) {% endif %} to skip the view instead. This is also the case with tables or views with [sequences]({% link {{ page.version.version }}/create-sequence.md %}), you will receive an error in the same way, or you need to pass [`skip_missing_sequences`]({% link {{ page.version.version }}/restore.md %}#skip-missing-sequences) with `RESTORE`.
```

Comment:
Here's the rendered preview for this: https://deploy-preview-18225--cockroachdb-docs.netlify.app/docs/stable/restore#object-dependencies